pkg/curriculum/curriculum.go

```diff
@@ -70,10 +70,12 @@ func NewRouter(s rest.Server, store Store) *chi.Mux {
 		// TODO: This is not used, refactor after curriculum got e2e test.
 		//r.Method("PATCH", "/subjects/{subjectId}", server.updateSubject())
 		r.Method("DELETE", "/", server.deleteSubject())
-		r.Method("GET", "/materials", server.getSubjectMaterials())
-		r.Method("POST", "/materials", server.createNewMaterial())
+		// Unused routes
+		//r.Method("GET", "/materials", server.getSubjectMaterials())
+		//r.Method("POST", "/materials", server.createNewMaterial())
```

Comment:
I think we could leave GET and POST uncommented inside here. Those are parts of the `/subject` route. And GET is being used on the curriculum page.